src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -99,153 +104,236 @@
 LOG_LINES = [{"is_error": 0, "line": f"This is line {i}"} for i in range(MAX_LOG_LINES)]
 
 
-def make_comm_msg(msg_type: str, job_id_like, as_job_request: bool, content: dict = None):
-    if content is None:
-        content = {}
-    job_arguments = {}
-    if isinstance(job_id_like, dict):
-        job_arguments = job_id_like
-    elif isinstance(job_id_like, list):
-        job_arguments[JOB_ID_LIST] = job_id_like
-    elif job_id_like:
-        job_arguments[JOB_ID] = job_id_like
-
-    msg = {
-        "msg_id": "some_id",
-        "content": {"data": {"request_type": msg_type, **job_arguments, **content}},
+# # Fixture to set and unset environment variable
+# @pytest.fixture(autouse=True)
+# def set_env_var(monkeypatch, job_test_wsname: str) -> None:
+#     """Set the KB_WORKSPACE_ID env var for all job comm tests."""
+#     # Set the environment variable; monkey patch automatically removes it afterwards
+#     monkeypatch.setenv("KB_WORKSPACE_ID", job_test_wsname)
+
+
+def check_error_message(
```

Comment:
moved all the check_* functions out of the obj, ready for de-objecting this workflow.d

Reply:
the functions haven't changed, other than adding docs and typing if they weren't already present